### PR TITLE
Add iat and nbf to Device Binding jws payload

### PR DESCRIPTION
**Pull request by vahancouver**
# JIRA Ticket

[SDKS-2748](https://bugster.forgerock.org/jira/browse/SDKS-2748)
# Description

[iOS] Add iat and nbf claims for Device Binding and Design Signing signed JWT

### CODE REVIEW

File: FRDeviceBinding/FRDeviceBindingTests/ApplicationPinDeviceAuthenticatorTests.swift [hunk 1]
```diff
@@ -49,6 +49,8 @@ class ApplicationPinDeviceAuthenticatorTests: FRBaseTestCase {
             XCTAssertEqual(messageDictionary[DBConstants.sub] as? String, userId)
             XCTAssertEqual(messageDictionary[DBConstants.exp] as? Int, Int(expiration.timeIntervalSince1970))
             XCTAssertEqual(messageDictionary[DBConstants.platform] as? String, DBConstants.ios)
+            XCTAssertGreaterThan(messageDictionary[DBConstants.iat] as! Int, Int(Date().timeIntervalSince1970) - 10)
```
Comment (jeyanthanperiyasamy) [inline FRDeviceBinding/FRDeviceBindingTests/ApplicationPinDeviceAuthenticatorTests.swift:52]: will this cause any flakky test ? just making sure,  may be another idea is to use XCTAssertEqualWithAccuracy

**Review by jeyanthanperiyasamy — COMMENTED** [review]
reviewed, looks good , just a minor comment

File: FRDeviceBinding/FRDeviceBindingTests/ApplicationPinDeviceAuthenticatorTests.swift (hunk 1 above)
Comment (vahancouver) [reply FRDeviceBinding/FRDeviceBindingTests/ApplicationPinDeviceAuthenticatorTests.swift]: @jeyanthanperiyasamy Good suggestion, changed to use XCTAssertEqualWithAccuracy

**Review by jeyanthanperiyasamy — APPROVED** [review]

**Review by spetrov — APPROVED** [review]
👍🏻

**Pull request merged**